### PR TITLE
Update use case response/request models

### PR DESCRIPTION
**Pull request by HarveyRDonnelly**
(i) Updated response/request models for the submit solution document, submit test document, register, login and logout use cases;
(ii) Added context variables to main view model;

### CODE REVIEW

File: src/main/java/driver/Main.java [hunk 1]
```diff
@@ -52,7 +53,7 @@ public static void main(String[] args) {
 
         Properties config = new Properties();
 
-        // Welcome Message :D
+        // Wel9come Message :D
```
Comment (darlynnguyen) [inline src/main/java/driver/Main.java:56]: Wel9come :D

**Review by darlynnguyen — APPROVED** [review]
Looks good to me 👍 !!

File: src/main/java/driver/Main.java (hunk 1 above)
Comment (N1cholasMcKee) [reply src/main/java/driver/Main.java:56]: I guess we're keeping this?

**Review by takedakento — APPROVED** [review]
Thanks Harvey! Well structured response/request models, Clean and up to date with the current view model

**Pull request merged**